code/game/machinery/medal_printer.dm

```diff
@@ -42,3 +43,12 @@
 	)
 	RefreshParts()
 
+/obj/machinery/r_n_d/attackby(var/obj/item/O as obj, var/mob/user as mob)
```

Comment:
remove `as obj`, `as mob`. It's not necessary

Reply:
fixed